var/spack/repos/builtin/packages/miniamr/package.py

```diff
@@ -13,10 +13,13 @@ class Miniamr(MakefilePackage):
     """
 
     homepage = "https://mantevo.org"
-    url      = "https://github.com/Mantevo/miniAMR/archive/v1.4.0.tar.gz"
+    git      = "https://github.com/Mantevo/miniAMR.git"
```

Comment:
I think you need both!

Reply:
why do we need both?

Reply:
One for the git checkout e.g. `branch="master"`, one of the tarball download.

Reply:
got it, done.